api/core/v2/flapping.go

```diff
@@ -12,7 +12,8 @@ func isFlapping(check *Check) bool {
 	}
 
 	// Is the check already flapping?
-	if check.State == EventFlappingState {
```

Comment:
This is why `low_flap_threshold` was broken before, as this function is called from inside `updateCheckState` but before `check.State` is updated (so it would've always been comparing `EventFlappingState` to an empty string).